roles/ngi_pipeline/templates/create_ngi_pipeline_dirs.sh.j2

```diff
@@ -17,7 +17,7 @@ fi
 mkdir -p {{ proj_root }}/$1/private/ngi_pipeline/log
 mkdir -p {{ proj_root }}/$1/private/ngi_pipeline/db
```

Comment:
If we're gonna re-make the log and db structure then it should be done properly, and these two needs changing as well. I.e. we want then to have something like 

```
    ../private/log/ngi_pipeline.log
    ../private/log/arteria/
    ../private/log/supervisord/
    ../private/log/taca.log
    ../private/log/tarzan/
    ../private/db/records_db_upps.sql
    ../private/db/tarzan/
```

because there are a bunch of places where these paths gets used: 

```
    (ansible-env) [johanhe@irma3 johanhe]$ egrep -Re 'ngi_pipeline_upps_path|ngi_pipeline_sthlm_path' 
    host_vars/127.0.0.1/main.yml:ngi_pipeline_sthlm_path: "{{ proj_root }}/{{ ngi_pipeline_sthlm_delivery }}/private/ngi_pipeline/"
    host_vars/127.0.0.1/main.yml:ngi_pipeline_upps_path: "{{ proj_root }}/{{ ngi_pipeline_upps_delivery }}/private/ngi_pipeline/"
    host_vars/127.0.0.1/main.yml:ngi_pipeline_log_sthlm: "{{ ngi_pipeline_sthlm_path }}/log/ngi_pipeline.log"
    host_vars/127.0.0.1/main.yml:ngi_pipeline_log_upps: "{{ ngi_pipeline_upps_path }}/log/ngi_pipeline.log"
    host_vars/127.0.0.1/main.yml:ngi_pipeline_db_upps: "{{ ngi_pipeline_upps_path }}/db/records_db_upps.sql"
    host_vars/127.0.0.1/main.yml:ngi_pipeline_db_sthlm: "{{ ngi_pipeline_sthlm_path }}/db/records_db_sthlm.sql"
    roles/func_accounts/defaults/main.yml:supervisord_log_dest_upps: "{{ ngi_pipeline_upps_path }}/log/supervisord/"
    roles/func_accounts/defaults/main.yml:supervisord_log_dest_sthlm: "{{ ngi_pipeline_sthlm_path }}/log/supervisord/"
    roles/arteria-siswrap-ws/defaults/main.yml:arteria_siswrap_log: "{{ ngi_pipeline_upps_path }}/log/arteria/siswrap-ws"
    roles/taca/templates/sthlm_fastq_delivery.yml.j2:    file: "{{ ngi_pipeline_sthlm_path }}/log/taca.log"
    roles/taca/tasks/main.yml:    ngi_pipeline_site_path: "{{ ngi_pipeline_upps_path }}"
    roles/taca/tasks/main.yml:    ngi_pipeline_site_path: "{{ ngi_pipeline_sthlm_path }}"
    roles/arteria-checksum-ws/defaults/main.yml:arteria_checksum_md5sum_log_dir: "{{ ngi_pipeline_upps_path }}/log/arteria/checksum_md5sum/"
    roles/arteria-checksum-ws/defaults/main.yml:arteria_checksum_service_log: "{{ ngi_pipeline_upps_path }}/log/arteria/checksum-ws/"
    roles/tarzan/defaults/main.yml:tarzan_log_dest: "{{ ngi_pipeline_upps_path }}/log/tarzan/"
    roles/tarzan/defaults/main.yml:tarzan_db_dest: "{{ ngi_pipeline_upps_path }}/db/tarzan/"
    roles/ngi_reports/defaults/main.yml:ngi_reports_log_sthlm: "{{ ngi_pipeline_sthlm_path }}/{{ ngi_reports_log }}"
    roles/ngi_reports/defaults/main.yml:ngi_reports_log_upps: "{{ ngi_pipeline_upps_path }}/{{ ngi_reports_log }}"
```

Also it seems kinda weird that the `ngi_reports` template has a variable called `log_dir` that is pointing to the `ngi_pipeline.log` path? 

```
    (ansible-env) [johanhe@irma3 johanhe]$ egrep -Re 'ngi_pipeline_log_upps|ngi_pipeline_log_sthlm' 
    host_vars/127.0.0.1/main.yml:ngi_pipeline_log_sthlm: "{{ ngi_pipeline_sthlm_path }}/log/ngi_pipeline.log"
    host_vars/127.0.0.1/main.yml:ngi_pipeline_log_upps: "{{ ngi_pipeline_upps_path }}/log/ngi_pipeline.log"
    roles/ngi_pipeline/tasks/main.yml:    ngi_pipeline_log: "{{ ngi_pipeline_log_upps }}"
    roles/ngi_pipeline/tasks/main.yml:    ngi_pipeline_log: "{{ ngi_pipeline_log_sthlm }}"
    roles/ngi_reports/templates/ngi_reports_sthlm.conf.j2:log_dir: {{ ngi_pipeline_log_sthlm }}
    roles/ngi_reports/templates/ngi_reports_upps.conf.j2:log_dir: {{ ngi_pipeline_log_upps }}
```